### PR TITLE
Improve utils performance and code style

### PR DESCRIPTION
**Pull request by nandordudas**
Refactor sensitive parts.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -1,39 +1,55 @@
-
-// Thanks @stimulus:
-// https://github.com/stimulusjs/stimulus/blob/master/packages/%40stimulus/core/src/application.ts
+/**
+ * @return {void}
+ */
+const __noop = () => { }
```
Comment (nandordudas) [inline src/utils.js]: It can be `export const __noop = () => { }` and you can import it.

File: src/utils.js [hunk 2]
```diff
@@ -190,7 +228,7 @@ export function transitionHelperOut(el, modifiers, settingBothSidesOfTransition,
     // use that.
     const duration = settingBothSidesOfTransition
         ? modifierValue(modifiers, 'duration', 150)
-        : modifierValue(modifiers, 'duration', 150) / 2
+        : modifierValue(modifiers, 'duration', 150) * 0.5
 
     const styleValues = {
         duration: duration,
```
Comment (nandordudas) [inline src/utils.js]: `dureation: duration,` => `dureation,`

**Comment by calebporzio** [comment]
For the same reason, I closed the last 2 PRs, I'm closing this one.

However, I'm interested in which parts of this PR is a "performance improvement" (would definitely be interested in those). Thanks!

**Comment by nandordudas** [comment]
Thank you for your patience, i will be more prudent!

**Pull request closed without merging**